src/TropicalGeometry/hypersurface.jl

```diff
@@ -260,3 +260,69 @@ end
 #     return get_attribute(TropH,:tropical_semiring_map)
 # end
+#
+
+@doc raw"""
+   tropical_prevariety(F::Vector{MPolyRingElem},nu::TropicalSemiringMap)
+
+Return the tropical prevariety generated by intersecting tropical hypersurfaces corresponding to elements of `F`.
+
+If `F` is a finite collection of polynomials with coefficients from a given field, return the tropical prevariety obtained by tropicalizing elements of `F` with respect to a given tropicalization map `nu`.
+
+If no `nu` is given, default to trivial valuation with `min` convention.
+
+If `F` is a collection of tropical polynomials, the function computes and intersects the associated hypersurfaces.
+# Example
+We compute the Dressian $\text{Dr}(2,5)$ below.
+```jldoctest
+julia> Gr25 = grassmann_pluecker_ideal(2,5)
+Ideal generated by
+  x[[1, 2]]*x[[3, 4]] - x[[1, 3]]*x[[2, 4]] + x[[1, 4]]*x[[2, 3]]
+  x[[1, 2]]*x[[3, 5]] - x[[1, 3]]*x[[2, 5]] + x[[1, 5]]*x[[2, 3]]
+  x[[1, 2]]*x[[4, 5]] - x[[1, 4]]*x[[2, 5]] + x[[1, 5]]*x[[2, 4]]
+  x[[1, 3]]*x[[4, 5]] - x[[1, 4]]*x[[3, 5]] + x[[1, 5]]*x[[3, 4]]
+  x[[2, 3]]*x[[4, 5]] - x[[2, 4]]*x[[3, 5]] + x[[2, 5]]*x[[3, 4]]
+
+#Compute Dressian without specified tropicalization map.
+julia> Dr25 = tropical_prevariety(gens(Gr25))
+Polyhedral complex in ambient dimension 10
+
+julia> rays_modulo_lineality(Dr25)[1]
+10-element SubObjectIterator{RayVector{QQFieldElem}}:
+ [1, -1//3, -1//3, -1//3, -1//3, -1//3, -1//3, 1//3, 1//3, 1//3]
+ [1, 1, -1, -1, 1, -1, -1, -1, -1, 3]
+ [1, -1, 1, -1, -1, 1, -1, -1, 3, -1]
+ [1, -1, -1, 1, -1, -1, 1, 3, -1, -1]
+ [-1, 1, 1, -1, -1, -1, 3, 1, -1, -1]
+ [-1, -1, -1, 3, 1, 1, -1, 1, -1, -1]
+ [-1, 1, -1, 1, -1, 3, -1, -1, 1, -1]
+ [-1, 3, -1, -1, -1, 1, 1, -1, -1, 1]
+ [-1, -1, 3, -1, 1, -1, 1, -1, 1, -1]
+ [-1, -1, 1, 1, 3, -1, -1, -1, -1, 1]
+
+#Compute with respect to max convention
+julia> nu = tropical_semiring_map(QQ,max)
+Map into Max tropical semiring encoding the trivial valuation on Rational field
+
+julia> Dr25max = tropical_prevariety(gens(Gr25),nu)
+Polyhedral complex in ambient dimension 10
+
+#Give input as tropical polynomials
+Dr25 = tropical_prevariety(tropical_polynomial.([f for f in gens(Gr25)]))
+Polyhedral complex in ambient dimension 10
+```
+"""
+#Input array of tropical polynomials
+function tropical_prevariety(F::Vector{<:AbstractAlgebra.Generic.MPoly{TropicalSemiringElem{minmax}}}) where {minmax<:Union{typeof(min),typeof(max)}}
+    length(F) > 0 || error("Empty array.")
+    HTF = tropical_hypersurface.(F)
+    return reduce(common_refinement, [h.polyhedralComplex for h in HTF])
+end
+
+#Default to min convention
+function tropical_prevariety(F::Vector{<:MPolyRingElem}, nu::TropicalSemiringMap)
+   return tropical_prevariety(tropical_polynomial.(F,Ref(nu)))
+end
+
+#Default to min with trivial valuation
+tropical_prevariety(F::Vector{<:MPolyRingElem}) = tropical_prevariety(F, tropical_semiring_map(coefficient_ring(first(F))))
```

Comment:
```suggestion
function tropical_prevariety(F::Vector{<:MPolyRingElem}, nu::TropicalSemiringMap)
   return tropical_prevariety(tropical_polynomial.(F,Ref(nu)))
end

#Default to min with trivial valuation
tropical_prevariety(F::Vector{<:MPolyRingElem}) = tropical_prevariety(F, tropical_semiring_map(coefficient_ring(first(F))))

tropical_prevariety(F::Vector{<:MPolyRingElem}, convention::minmax) where {minmax<:Union{typeof(min),typeof(max)}} = tropical_prevariety(F, tropical_semiring_map(coefficient_ring(first(F)), convention))
```

I suggest to add a shortcut for also just specifying `min` or `max` without having to construct a `TropicalSemiringMap`. Does that make sense? Otherwise I am fine with this.

I removed one of the comment lines that probably only was left over.

Reply:
Looks good to me!